sideci_demo/phpmd_demo.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use Github\Client;
+
+class Repository
+{
+    private $owner;
+    private $name;
+    private $public;
+    private $branches;
+
+    const PublicVisibility = 'public';
```

Comment:
__[PHP_CodeSniffer]__

Class constants must be uppercase; expected PUBLICVISIBILITY but found PublicVisibility


<!-- Comment by [Sider](https://sider.review) -->